libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -157,6 +157,8 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
 
   @HostListener('window:keyboardWillShow', ['$event'])
   _onKeyboardWillShow(info?: { keyboardHeight: number }) {
+    console.log('_onKeyboardWillShow', info);
+
```

Comment:
I don't think we want `console.log`s in our widgets... please remove.

Reply:
Hahaha... typisk mig at jeg fik sneget den med ind.